src/marshmallow/fields.py

```diff
@@ -1218,25 +1218,32 @@ class DateTime(Field):
     Example: ``'2014-12-22T03:12:58.019077+00:00'``
 
     :param format: Either ``"rfc"`` (for RFC822), ``"iso"`` (for ISO8601),
-        or a date format string. If `None`, defaults to "iso".
+        ``"timestamp"``, ``"timestamp_ms"`` (for a POSIX timestamp) or a date format string.
```

Comment:
Wondering (out loud) about the choices and their naming. I don't use timestamps. Are those two the two common uses?

`timestamp` is a number of seconds. Should we call it `timestamp_s`? I don't think so. Why the need for `timestamp_ms` (and not other units)? Is it also commonly used?

Reply:
Thanks for the review! Good question. These are the two common use cases (to my knowledge). JavaScript tends to use timestamps in milliseconds, while Python uses seconds by default.